rotkehlchen/chain/evm/decoding/paraswap/decoder.py

```diff
@@ -195,9 +195,13 @@ def addresses_to_decoders(self) -> dict[ChecksumEvmAddress, tuple[Any, ...]]:
         }
 
     def decoding_by_input_data(self) -> dict[bytes, dict[bytes, Callable]]:
-        return {  # swapOnUniswapV2Fork, swapOnUniswapV2ForkWithPermit, buyOnUniswapV2Fork
+        return {
             method_id: {SWAP_SIGNATURE: self._decode_uniswap_v2_swap}
-            for method_id in (b'\x0b\x86\xa4\xc1', b'n\x91S\x8b', b'\xb2\xf1\xe6\xdb')
+            for method_id in (
+                b'\x0b\x86\xa4\xc1', b'\xf5f\x104',  # swapOnUniswapV2Fork and variant using factory  # noqa: E501
+                b'n\x91S\x8b',  # swapOnUniswapV2ForkWithPermit
+                b'\xb2\xf1\xe6\xdb',  # buyOnUniswapV2Fork
```

Comment:
why not make those into `FINAL` constants in this module, with the name essentially being the function. This way no need for comments. Same for other such cases in this file.